graph/tests/.latest.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Should this file be added to `.gitignore`? When running the tests, this got updated automatically

Reply:
added to `.gitignore`